engine/suspender.go

```diff
@@ -357,7 +357,16 @@ func (eng *Engine) Suspender(ctx context.Context, cs *kubernetes.Clientset) {
 
 			// now we can check if patchedResourcesCounter is > 0 and add nextSuspendTime depending of the result
 			if patchedResourcesCounter > 0 {
-				sLogger.Debug().Str("step", stepName).Msgf("namespace has been unsuspended manually, adding the annotation '%s' to it (engined configured duration: '%s')", eng.Options.Prefix+NextSuspendTime, eng.RunningDuration)
+				sLogger.Debug().Str("step", stepName).Msg("namespace has been unsuspended manually")
+
+				sLogger.Debug().Str("step", stepName).Msgf("checking annotation '%s'", eng.Options.Prefix+NextSuspendTime)
+				if val, ok := n.Annotations[eng.Options.Prefix+NextSuspendTime]; ok {
+					sLogger.Info().Str("step", stepName).Msgf("found annotation '%s'='%s', not updating it", eng.Options.Prefix+NextSuspendTime, val)
+					continue
```

Comment:
:hammer_and_wrench: this will skip the log on line 398

Reply:
Updated